### ISSUE TITLE
🐛 No errors appear when biome.json has parse errors

### ISSUE THREAD

**Issue by olsonpm** [biomejs/biome-vscode#1113]
### VS Code version

1.126.04524

### Extension version

 3.7.1

### Biome version

2.5.5

### Operating system

- [ ] Windows
- [ ] macOS
- [x] Linux

### Description

Biome doesn't report any errors when the config can't be parsed.

Combined with the setting `"biome.requireConfiguration": true`, this results in a non-functioning Biome plugin with no indication to why.

### Steps to reproduce

Minimal reproduction at [repro/biome-vscode](https://github.com/olsonpm/repro/tree/biome-vscode)

1. Clone

```sh
$ git clone -b biome-vscode git@github.com:olsonpm/repro.git
$ cd repro
```

2. Open vscode

```sh
$ code .
```

3. Confirm issue

    a. You can't format test.js e.g. Format With -> Biome option doesn't exist

    b. No errors appear in the Biome output

### Expected behavior

If the configuration file can't be parsed, then I would expect that to show up as an error or warning in the Biome output

### Does this issue occur when using the CLI directly?

The CLI reports the config parsing error - here's the CLI output from the repro

<details>

<summary>show output</summary>

```
❯ npx biome check
/path/to/repro/biome.json:31:4 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Expected a property but instead found '}'.
  
    29 │ 			"source": {
    30 │ 				"organizeImports": "on",
  > 31 │ 			}
       │ 			^
    32 │ 		}
    33 │ 	}
  
  ℹ Expected a property here.
  
    29 │ 			"source": {
    30 │ 				"organizeImports": "on",
  > 31 │ 			}
       │ 			^
    32 │ 		}
    33 │ 	}
  

configuration ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Biome exited because the configuration resulted in errors. Please fix them.
```

</details>

### Link to a minimal reproduction

https://github.com/olsonpm/repro/tree/biome-vscode

### Logs

I don't see a Biome Trace category

[Here are the biome rage logs](https://paste.to/?c5d0ddf8cfb7da2c#HdMmM31VhBm9YkGtz2Kdm3eTAYhEuwNWb3VKGWfEdb8T) - using password 'biome-rage-output'

**Comment by ematipico** [comment]
We need to know if you see the same issue in the CLI. That's not an optional question.

**Comment by olsonpm** [comment]
the issue has nothing to do with the CLI.  It has to do with this library's lack of error reporting.

and I answered the question correctly "Not applicable".  I didn't treat it as optional.

**Comment by ematipico** [comment]
We need to you know if you can reproduce the same issue with CLI so we know where's the bug and how to fix it.

**Comment by olsonpm** [comment]
Okay, here's the biome cli output.  I'll update the original post template with this

<details>

<summary>show output</summary>

```
❯ npx biome check
/path/to/repro/biome.json:31:4 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Expected a property but instead found '}'.
  
    29 │ 			"source": {
    30 │ 				"organizeImports": "on",
  > 31 │ 			}
       │ 			^
    32 │ 		}
    33 │ 	}
  
  ℹ Expected a property here.
  
    29 │ 			"source": {
    30 │ 				"organizeImports": "on",
  > 31 │ 			}
       │ 			^
    32 │ 		}
    33 │ 	}
  

configuration ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Biome exited because the configuration resulted in errors. Please fix them.
```

</details>
